tests/testproject/TestProject.AppHost/TestProgram.cs

```diff
@@ -32,12 +32,24 @@ private TestProgram(string[] args, Assembly assembly, bool includeIntegrationSer
 
         if (includeIntegrationServices)
         {
-            var sqlserverContainer = AppBuilder.AddSqlServerContainer("sqlservercontainer");
-            var mysqlContainer = AppBuilder.AddMySqlContainer("mysqlcontainer");
+            var sqlserverDbName = "tempdb";
+            var mysqlDbName = "mysqldb";
+            var postgresDbName = "postgresdb";
+            var mongoDbName = "mymongodb";
+
+            var sqlserverContainer = AppBuilder.AddSqlServerContainer("sqlservercontainer")
+                .AddDatabase(sqlserverDbName);
+            var mysqlContainer = AppBuilder.AddMySqlContainer("mysqlcontainer")
+                .WithEnvironment("MYSQL_DATABASE", mysqlDbName)
+                .AddDatabase(mysqlDbName);
             var redisContainer = AppBuilder.AddRedisContainer("rediscontainer");
-            var postgresContainer = AppBuilder.AddPostgresContainer("postgrescontainer");
+            var postgresContainer = AppBuilder.AddPostgresContainer("postgrescontainer")
+                .WithEnvironment("POSTGRES_DB", postgresDbName)
+                .AddDatabase(postgresDbName);
             var rabbitmqContainer = AppBuilder.AddRabbitMQContainer("rabbitmqcontainer");
-            var mongodbContainer = AppBuilder.AddMongoDBContainer("mongodbcontainer");
+            var mongodbContainer = AppBuilder.AddMongoDBContainer("mongodbcontainer")
+                .AddDatabase(mongoDbName);
+
             var sqlserverAbstract = AppBuilder.AddSqlServerContainer("sqlserverabstract");
```

Comment:
@mitchdenny - what are these "Abstract" containers? It looks to be doing the same thing as above - `AppBuilder.AddSqlServerContainer`, just with a different name.